### PR TITLE
Create common.json

### PR DESCRIPTION
**Pull request by MrMarioMichel**
A german Translation

### CODE REVIEW

File: src/i18n/locales/de-DE/common.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "title": "Emerald - Ethereum Classic Wallet", // Emerald - Ethereum Classic Wallet
+    "menu": {
+        "home": "Home",
+        "addressBook": "Adressbuch", // Adress Book
+        "contracts": "Contracts", // Contracts
```
Comment (OpenMindedSource) [inline src/i18n/locales/de-DE/common.json]: Verträge

File: src/i18n/locales/de-DE/common.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        "network": "Netzwerk" // Network
+    },
+    "submit": "einreichen", // Submit
```
Comment (OpenMindedSource) [inline src/i18n/locales/de-DE/common.json]: "Übernehmen" maybe

Comment (MrMarioMichel) [reply src/i18n/locales/de-DE/common.json]: Yeah that fits

File: src/i18n/locales/de-DE/common.json (hunk 1 above)
Comment (MrMarioMichel) [reply src/i18n/locales/de-DE/common.json]: Yeah forgot that one

**Comment by BelfordZ** [comment]
@MrMarioMichel what do you want to do with this PR?

**Pull request closed without merging**